### PR TITLE
add tag component and use in OpenSessionOverview

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: src/OpenSessionOverview/OpenSessionOverview.jsx [hunk 1]
```diff
@@ -111,7 +112,7 @@ class OpenSessionOverview extends React.Component {
             {!!tags.length &&
               <div className="session-tags">
                 {tags.map(
-                  (t, idx) => <div className="topic overview-tag" key={idx}>{t}</div>)}
+                  (t, idx) => <Tag key={idx} displayName={t} {...this.props}/>)}
```
Comment (tholex) [inline src/OpenSessionOverview/OpenSessionOverview.jsx]: `this` has a ton of props, why pass them into the tag?

Comment (learykara) [inline src/OpenSessionOverview/OpenSessionOverview.jsx]: you're right - updated to just pass config thru

**Comment by learykara** [comment]
@tholex ok to merge?

**Comment by tholex** [comment]
:+1:

**Pull request merged**